pages/integrations/amazon_eventbridge.md

```diff
@@ -17,7 +17,7 @@ Once you've configured an Amazon EventBridge notification service in Buildkite,
   <tr><th><a href="#events-build-finished">Build Finished</a></th><td>A build has finished</td></tr>
   <tr><th><a href="#events-job-scheduled">Job Scheduled</a></th><td>A job has been scheduled</td></tr>
   <tr><th><a href="#events-job-started">Job Started</a></th><td>A command step job has started running on an agent</td></tr>
-  <tr><th><a href="#events-job-finished">Job Finished</a></th><td>A job has finished</td></tr>
+  <tr><th><a href="#events-job-finished">Job Finished</a></th><td>A job has finished. To check a job's result, use the `passed` field. The value is `true` when the job passed, and `false` otherwise.</td></tr>
```

Comment:
Since this is in HTML, the Markdown syntax doesn't render properly:

![image](https://user-images.githubusercontent.com/119824349/226813815-06081f5f-5a74-4786-914f-25cfad796e8a.png)

```suggestion
  <tr><th><a href="#events-job-finished">Job Finished</a></th><td>A job has finished. To check a job's result, use the <code>passed</code> field. The value is <code>true</code> when the job passed, and <code>false</code> otherwise.</td></tr>
```